body.hbs

```diff
@@ -36,6 +36,6 @@
   </section>
 
   <p>
-  The results of the tests are show below:
+  The results of the tests are shown below:
```

Comment:
nice. it looks like this typo is in my most of the test suites: https://github.com/search?q=org%3Aw3c-ccg+show+below&type=code